packages/client/src/components/Ammunition/Ammunition.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react'
+
+type AmmunitionProps = {
+  ammun: number
+}
+
+export const Ammunition: FC<AmmunitionProps> = value => {
+  const list = []
+
+  for (let i = 0; i < value.ammun; i++) {
+    list.push(<li key={i} className="ammunition__item" />)
+  }
+
+  return <ul className="ammunition">{list}</ul>
+}
```

Comment:
Not required, just a suggestion: 
```suggestion
export const Ammunition: FC<AmmunitionProps> = ({ ammun }) => {
  const list = [...Array(ammun).keys()];

  return (
      <ul className="ammunition">
        {list.map((el) => (
            <li key={el} className="ammunition__item" /> 
        ))}
      </ul>
  );
};

```

Reply:
Спасибо, поправил, твое решение лучше и красивее будет)

Reply:
Сделано